src/containers/Explorer/index.js

```diff
@@ -28,13 +38,15 @@ class Explorer extends React.Component {
       history,
       resetFilters,
       load,
+      addPlatformFilter,
       toggleStageFilter
     } = this.props;
 
-    if (prevProps.locationNonce !== locationNonce && history.action === 'POP') {
+    if (prevProps.locationNonce !== locationNonce) {
```

Comment:
any particular reason for the `&& history.action === 'POP'`? it seems like it was causing the component to not reset if the same page is clicked that the user is already on